webapp/src/pages/Dashboard.tsx

```diff
@@ -71,7 +71,9 @@ const Dashboard = () => {
           color="secondary"
           variant="contained"
           component={Link}
-          to={`/${jobId}/dataset_splits/${firstAvailableDatasetSplit}/prediction_overview${searchString}`}
+          to={`/${jobId}/dataset_splits/${firstAvailableDatasetSplit}/${
+            !isPipelineSelected(pipeline) ? `utterances` : `prediction_overview`
```

Comment:
I find the condition is easier to read if you remove the `!` and flip. Also, regular strings would be goof enough here.
```suggestion
            isPipelineSelected(pipeline) ? "prediction_overview" : "utterances"
```

Reply:
Sure. done